.github/workflows/hexpm-mirrors.yml

```diff
@@ -0,0 +1,58 @@
+---
+name: hexpm-mirrors
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  test_failing_first_mirror:
+    name: Test option hexpm-mirrors with a dummy first mirror
+    runs-on: ${{matrix.os}}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: ['ubuntu-22.04', 'windows-2022']
+        otp-version: ['24']
+        elixir-version: ['v1.14', '']
+        install-rebar: [true, false]
+        install-hex: [true, false]
+    steps:
+      - uses: actions/checkout@v3
+      - name: Use erlef/setup-beam
+        id: setup-beam
+        uses: ./
+        with:
+          otp-version: ${{matrix.otp-version}}
+          elixir-version: ${{matrix.elixir-version}}
+          install-rebar: ${{matrix.install-rebar}}
+          install-hex: ${{matrix.install-hex}}
+          hexpm-mirrors: |
+            https://mirror.invalid
+            https://cdn.jsdelivr.net/hex
+            https://repo.hex.pm
```

Comment:
Not required, if it's the last one, but having it for clarity should not change the final list (which would only append if not present).